README.md

```diff
@@ -1,192 +1,74 @@
-This repo provides examples about how to use GPU powered XGBoost-Spark to train XGBoost model and use it to do predictions. 
+RAPIDS.ai XGBoost-Spark Apache Spark Examples
+=============================================
 
-Then content mainly contains：
+This repo provides example applications that demonstrate the RAPIDS.ai GPU-accelerated XGBoost-Spark project.
 
-  * [Prepare Example Jar](#Prepare-Example-Jar)
-  * [Prepare Dataset](#Prepare-Dataset)
-  * [Run Example App](#Run-Example-App)
+Table of contents:
 
+  * [Build Examples Jar](#Build-Examples-Jar)
+  * [Getting Started Guides](#Getting-Started-Guides)
+  * [Other Examples](#Other-Examples)
+  * [Preparing Datasets](#Preparing-Datasets)
 
-# Prepare Example Jar
+# Build Examples Jar
 We use [maven](https://maven.apache.org/) to build jar package.
 
 Our example relies on [cuDF](https://github.com/rapidsai/cudf) and [XGBoost](https://github.com/rapidsai/xgboost)
```

Comment:
do we want to update the xgboost link to be our branch: https://github.com/rapidsai/xgboost/tree/rapids-spark

Reply:
good catch, done.